### DISCUSSION TITLE
PlayStation Version Changes

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#144] [General]
A kind of sister topic to the level conversions (err, all three of them), collecting the differences of the PlayStation version, with an eye for possibly porting some stuff back

The obvious stuff that would be hard to port :

- Totally different lighting system
- Vehicles have faked headlights (not like the ones in the pre-alpha footage with "real" lights that was abandoned)
- Banks have a new design with a new texture of a dollar sign on flags to make it clear what the purpose of the building is

Stuff that could be fair game:

- New sound effects for some weapons. The sounds for the Uzi and minigun are nice with a bit more bite to them. The uzi sound is actually used in Theme Hospital too, so would serve as a high quality source (SW version is sped up and cut). Most of the other sound effects are a) the same and b) let down by the atrocious sample rate used - the female scream sound is like it was recorded down a bad phone line.
- Levels. There are only three new levels, a single eurocorp and zealot level, and a test level. I've converted them all already. There do seem to be some small differences on some levels, mainly due to the lack of trains.
- The cerberus IFF has a new graphic. I've personally always hated the one in the DOS original, it just looks like a gas cylinder or something. The PS1 graphic isn't great either, but at least looks somewhat like the thing show in the equip screen:
![](https://i.ibb.co/1QrTw7n/SLUS-002-62-09022024-210010-0252.png)
I went to rip this, but there's so much crammed into the graphics I've actually not even found it yet.
- Some new map textures. I intend to port these (95% are the same as the DOS one, so battling with the Colour Look Up Tables isn't so painful), but we need a tool to save files like .pngs in the final .DAT raw format (there must be a tool for Dungeon Keeper?)

Misc:

- The moon level has a mech in it! It's only the claw mech, not the DOS unique model, but it starts in the same place, and notably can escape from the area it is in. Video of this here: https://youtu.be/fjr42m2oR7w?list=PLiUlAEB9Jcu-fdMFMaEHyyrpT0LQRqhep&t=167
- It's possible to buy the epidermis items, as discovered by @mefistotelis. I'm not sure this is actually normally possible, but there's a video of it running with various hacks to unlock them here: https://youtu.be/ZFPpvZI8o5o?list=PLiUlAEB9Jcu-fdMFMaEHyyrpT0LQRqhep&t=20

Some quite nice loading screens, although they all depict scenes not possible in the actual game, from the PS1 intro:

![](https://i.ibb.co/5ctBLcT/loadscreen1.png)
![](https://i.ibb.co/jJmPjFQ/loadscreen2.png)
![](https://i.ibb.co/TWZYkxR/loadscreen3.png)
![](https://i.ibb.co/yyzpS83/loadscreen4.png)

Oh, a weird thing, the console port famously doesn't have the scientists in-game (they just use briefcase men as stand-ins). However, they ARE in the graphic files and are loaded into VRAM on level start. The Shady Guys and some of the other missing civilians are in there too. Weird that they were cut, possibly not all frames are there, but a lot are. Considering a load of space is used up by the old HUD that was scrapped, they probably had enough VRAM for all of them if they cleaned up a bit.

**Comment by Moburma** [comment]
Ripped the PS1 Cerberus IFF. Talk about finding a needle in a haystack...

![](https://i.ibb.co/Rcxvpby/Cerberus-IFF-Ps1.png)

**Comment by Moburma** [comment]
Ported over some of the PS1 textures. Not sure this is a particularly worthwhile endeavour, but hey. 
The big problem is the PS1 version runs in 16 bit colour, and stores its textures using Colour Look Up Tables. This means each tiny sub-graphic in the texture sheet has its own 16 colour palette. In the PC game it only runs in 8 bit colour, so all the texures use the same 256 colour palette. Therefore the PS1 textures can have colours simply not in the DOS one. I've just let GIMP map them over to the PC game palette, but they look somewhat washed out as a result. Note the inexplicable reappearance of the Indian man poster from the original Syndicate, the new neon signs and the pointless change for the red metal wall to say "PROPERTY BULLFROG" instead of "PROPERTY OF KIKOHO". Something else I never noticed is that the kind of white window with red supports across it actually has a silhouette of saucy ladies on it in the original, but is just plain white here.

![](https://i.ibb.co/4JtLrMK/PS1TEX01.png)

**Reply by mefistotelis** [reply]
For KeeperFX, I made a tool which creates texture atlas from 32-bit PNGs. It could be used for SW Port as well.

The tool gets a bunch of PNGs with square textures merged into reasonable images, divides and shuffles the squares at proper positions. Here you can see the input files:
https://github.com/dkfans/FXGraphics/tree/main/enginefx/textures-128

Good to know the PS1 version has more colors, this could be later used to improve quality of PC textures.

**Reply by Moburma** [reply]
Yes, I actually discovered this tool thanks to the KeeperFX people last night! It does indeed work, but I've had some odd issues with colours and transparency so need to play with it to understand the parameters needed for SW. I got the above textures working in-game (with some weird effects with the white lights though) and also the unused set of Japanese textures from the JP version (https://tcrf.net/Syndicate_Wars_(DOS)/Japanese_Version#Main_Changes). 
In terms of colours, the PS1 textures are swings and roundabouts - while they can have more _different_ colours because of the CLUTs, in practice they are often simpler than the PC, and each individual texture (inside the atlas) potentially has less colours (only 16 versus 256). Also I cheated above by just pasting the changed ones over the PC ones, but actually there may be more changes I haven't noticed for the common ones. I haven't worked out how to automatically rip the textures, I'm doing this by hand using this: https://github.com/romhack/PsxVram-DotNet
Which works, but is extremely time consuming and manual (and you have to have an idea how each texture should look to know if you have the right palette!).

Haven't tried your tool for the MSPR files yet.

**Comment by Moburma** [comment]
Something I played around with, I recreated the PS1 Eurocorp campaign.

https://pastebin.com/3rkJbT7V

Save it over miss000.ini to use.

This has no research, you just get awarded new items after each level. There are also much fewer levels and it's totally linear, you don't get any choice of level, you simply complete one level then get sent on the next.
This didn't work that great though, currently SW only supports awarding one item after each mission only, when the PS1 version commonly awards two. As such you don't get the body mods awarded, making this rather useless. Also because the research centre deliberately isn't rewarded after the first level you don't get research notifications after a level to know what you've unlocked. 
My port of the first Eurocorp mission is in place, but it has buggy netscan info as there isn't one for this new mission and it's trying to use the ones for the original London level when actually the new level is actually set in Vancouver (the game recycles the same briefing and pretends it's London).

As such this kind of a dead end and doesn't really work very well, but I thought I would share it as a curiousity. Doesn't seem worth "fixing" the above for a novelty campaign that is worse than the original game.

It also made me realise why they redid the first level - the London map actually is not present at all in the final PS1 game. They clearly had big issues getting it to work in the console version, and the Duplicity levels are missing as well as a result. In fact it dumps the outcome of that level into the follow up save Drennan missions, so that is one of the few places where the briefing is different.
The different version of London in the Korean version likely shows an attempt at getting the level working on the PS1, which is why they took all the walls out, I assume they used up a lot of Thing slots, or slowed down performance, etc.

**Comment by Moburma** [comment]
Another observation on the PS1 version. The final mission(s) has an extra objective not present in the PC original. Once the Nine are defeated, the player's units must move towards part of the zealot superweapon to "evacuate", the level doesn't just end once all enemies are dead.

**Reply by mefistotelis** [reply]
Hm, my idea after seeing the outro was that the agents stayed stranded on the moon.
(though in the menu after outro they are available..)

**Reply by Moburma** [reply]
Yes, it's a bit weird and contradicted by the ending video. Also you're going inside the superweapon thing.... what's even going to be inside it? 
See here: https://youtu.be/fjr42m2oR7w?list=PLiUlAEB9Jcu-fdMFMaEHyyrpT0LQRqhep&t=224

The PS1 version doesn't have the credits or even the final epilogue text, so if you wanted to know the significance of what just happened, tough!

**Reply by mefistotelis** [reply]
> what's even going to be inside it?

It's supposed to be weather control system. Definitely not something you can use to go back, for that they'd have to wait for a shuttle. But I guess they could wait inside, where they're probably shielded from sun radiation, and have breathable air.

**Reply by Moburma** [reply]
But would anyone even bother to get them? It's not made clear if agents have any kind of personality or real autonomy left (outside of combat), they just seem to basically be terminator-esque robots controlled by remote. As such I'd imagine evil megacorp Eurocorp would have no sentimental attachment to them and happily just leave them on the moon if it cost too much to get them down again. 
God only knows what is meant to happen next after the zealot ending.

**Reply by mefistotelis** [reply]
If there's a weapon on the moon, and an evil cult wants to get it, having a group of agents there would actually be logical.

So maybe on PS1 they're going in because they're staying?

**Comment by Moburma** [comment]
A little bit off topic, but looking in the Sega Saturn magazine coverage of the Saturn game they have some CGI images that aren't actually in the final PS1 version:


![](https://i.ibb.co/8c18hcg/policecar.jpg)
This one looks like it follows one from one of the actually used between level screens as shown in my first post.

![](https://i.ibb.co/wCSFnxf/city.jpg)

![](https://i.ibb.co/BTQL7cd/robots.jpg)

I noticed also the PS1 intro features the advert for Creation that was on the first beta demo of SW, with the footage of the real life dam.

**Reply by mefistotelis** [reply]
I remember someone sending me the two animations with trucks, but I can't remember when/how. The person did claimed it was from SW, but looking at the animations it seemed closer to Hi-Octane, if at all related. So I ignored it.

It was two very short animations (like 5 seconds), presenting the same scene of a truck closing in and stopping. The only difference between them was the lights, which were off in one, and lit in the other - like on the frames you've pasted.

Honestly, it looked like a garbage truck stopping to collect the load. Also somehow I didn't even noticed the figure standing there.

**Reply by Moburma** [reply]
Duh, I just checked the game again and that scene IS in the PS1 version, it's in the long new video of various space scenes. This whole thing is really weird, it's not very Syndicate Wars-y in many ways, but it does use the same vehicles as the PS1 intro so it's not just from something else. I do wonder if bits of it were lifted from Creation or even another unreleased game though. 
The Hi-Octane comparison is apt too, there is this weird overlap between the two games with the CGI police car (that doesn't appear in either game!). It's more like someone's side idea that was just crowbarred into both games because it looked good and they didn't have anything more appropriate created.

**Reply by mefistotelis** [reply]
So many flare effects. It did looked cool back in the day. I started hating flares only after that Star Trek movie which used at least one in every scene.

**Comment by Moburma** [comment]
A minor thing - I was able to rip the sound effects from the PS1 version using this:
https://github.com/vgmtrans/vgmtrans
However, it doesn't work great, and usually truncates the samples heavily. Mildly interesting to poke around though, the PS1 version uses an altered version of the second sound set, and includes sounds for all kinds of stuff cut from the PS1 version such as email notifications and interestingly the mech growling sound.

**Reply by Moburma** [reply]
I noticed a new version of https://github.com/vgmtrans/vgmtrans is out, and trying that I successfully ripped all the PS1 sound samples. These are mostly exactly the same (and sound terrible due to their pathetic sample rate), but a few are nice, especially the new Uzi sound effect. Thankfully they dropped right into place, so I will upload the changes to the sound repository (had some issues with the minigun sounds, it seems to need to play at a pathetic 8000hz, but the PC version won't stoop that low, so I need to change its pitch).

If you want to rip yourself then the samples are in the files beginning with FX in the F and G directories of the game (F being Front end - i.e. menus - and G being in-Game proper). Note the tool saves everything as 44,000hz, so the sample rates are wrong for the game. Any you want to play with need to be re-saved, some samples are at 11025hz, but others are at a really crappy 8000hz. Don't forget they resized the PC version's nearly 10MB sample file down to 450KB!. 

Here's the little mission complete jingle that is not in the PC version: 
https://vocaroo.com/14gydMEsIjsD

**Reply by Moburma** [reply]
Still not got the damned Minigun sounds working quite right yet, I've had to transform them in Audacity. I wonder if there is some kind of DSP effect on them in the PS1 version as they are bass-y and loud as hell there, but the sample on disc is very tame and sounds wrong on its own on PC. Anyway, I made a video of what they sound like so far:

https://www.youtube.com/watch?v=OkJ6uFs9ryg

**Reply by Moburma** [reply]
I uploaded the files now. To get the minigun sounds right I had to reduce speed 50%, then increased bass by a few dB (in Audacity). I think the PlayStation does some DSP filtering on the sounds, the minigun is very bass-y and loud there. I think actually it only needs the speed adjustment to be correct on PC, but it sounds extremely feeble and just wrong without the massive bass behind it, especially if all agents are firing, so I went Iggy Pop on it and cranked them just off red. Not quite right, but about as good as you're going to get comparing apples to oranges.

**Comment by Moburma** [comment]
Something I ripped ages ago but forgot about, the PS1 icons to choose Zealots or Eurocorp:

![](https://i.ibb.co/q0gG9jp/PS1COTNE.png)

![](https://i.ibb.co/ynQnKL2/PS1-Eurocorp.png)

Look a bit odd though, as they're saved as 2:1 ratio images due to the pseudo high res screen format used by the PS1 menus. Therefore you have to stretch them horizontally to be in proportion, but they lose detail stored like this.

**Reply by mefistotelis** [reply]
Yeah, I used one of these as book of cataclysm symbol:
http://syndicate.lubiki.pl/swars/html/swars_book_cataclysm.php

**Reply by Moburma** [reply]
Doh!

**Reply by Chonkblonk** [reply]
Oh, cool, you're the guy who runs that site? It's better than 99% of fan wikis, and I've always relied on it overall the years whenever I got the itch to play the game again, ever since I was a kid.

**Comment by Moburma** [comment]
Another rip:

![](https://i.ibb.co/XW6YRWq/PS1tree.png)
![](https://i.ibb.co/dfN9y0J/PS1tree-Burned.png)

This replaced the big round tree from the PC one (which does look a bit crap and out of place, I'm sure it's recycled from another Bullfrog game). You can see a comparison of them in this post [here](https://github.com/swfans/swars/discussions/141#discussion-6155157).

**Comment by Moburma** [comment]
I realised I hadn't gotten the destroyed image for the Cerberus IFF, so here are both again:

![](https://i.ibb.co/Rcxvpby/Cerberus-IFF-Ps1.png)
![](https://i.ibb.co/JkCNsS1/Cerberus-IFFDestroyed-Ps1.png)

I think the main things left are weapon/case designs. These are a pain to get as they use weird palettes and are tiny in the sprite sheets.

**Comment by Moburma** [comment]
A minor thing I just noticed, there's a small kind of L shaped building in the game, usually has a gun shop sign on it. In the PC version you can't actually go in this, the doors are just a texture on the wall. In the PS1 version this is fixed and it has a real door so you can actually go inside.

**Comment by Moburma** [comment]
There are no domes in the PS1 version, they are removed from all maps.

**Comment by Moburma** [comment]
Ripped this ages ago and forgot about it:

![](https://i.ibb.co/ZhfzbvB/Bank-Flag-Ps1.png)

This is hanging outside the banks in the console version.

**Comment by Chonkblonk** [comment]
Say, is there any kind of sprite work that you people need done for anything? I could try my hand at altering or drawing new ones from scratch.

**Reply by mefistotelis** [reply]
Having larger size sprites and textures would be nice.

We have a separate [repo for GFX assets](https://github.com/swfans/swars-gfx).
You can already find the simple things - like fonts and panels - there. If that's what you're able to improve, take a look.
Besides fixing the existing sprites, other sizes could be introduced. Folders have size in their names - for various sprite collections, size "32" means for 320x200, "64" - for 640x480 and so on. For fonts, and collections of sprites of same size (weapon icons) size is the font height.

If you're more ambitious, please let me know and I will start working on adding sprites for people and in-game objects, or terrain textures if you prefer to work on these.

The game can support sprites of size up to 255x255, and textures could be up to 256x256 (though the textures would require some more programming work to add). Sprites exceeding these bounds, ie. those in the hud panel, are possible because they can be cut and stores as several sub-sprites.

**Reply by Chonkblonk** [reply]
I don't think I could do much for scaling up any graphics, but I could maybe draw some new things, or perhaps redraw or alter something. I see that they're in a non-standard (to me) format, do I need to convert that to get access to it, or does it exist as .png sheets somewhere?

**Reply by mefistotelis** [reply]
> I don't think I could do much for scaling up any graphics

Well, that is most of the GFX work required on such old games.

Just to make sure we're on the same page in regard to what work is needed here - an example from my previous project, KeeperFX:
[original](https://github.com/dkfans/FXGraphics/blob/main/enginefx/textures-32/pack000/bookshelf_wooden.png) vs [rescaled](https://github.com/dkfans/FXGraphics/blob/main/enginefx/textures-160/pack000/bookshelf_wooden.png).
(this was done by a volunteer artist many years ago)

For new graphics - one thing I can think of is a menu background for Unguided Campaign.

> I see that they're in a non-standard (to me) format, do I need to convert that to get access to it, or does it exist as .png sheets somewhere?

There are extractors available for all formats. To get sprites, you mostly need DAT/TAB extractor.

As already mentioned, we have a separate [repo for GFX assets](https://github.com/swfans/swars-gfx).
It has files in PNG format. But it contains only the GFX files which we already use - so anything which is not modified in SW Port is not there.
I can put some specific files there earlier, if you have an idea on what you want to focus on.

**Comment by Moburma** [comment]
Some more rips:

![](https://i.ibb.co/rKqtYjpr/weaponfinal.png)

![](https://i.ibb.co/jc0F2PY/weapon.png)

The first one is the standard weapon graphic. It's a bit odd as I don't think the PS1 version has two graphics like the PC one, it only seems to use this.

However, this then leads to the second graphic above. I'm not sure this is even actually used, and also not sure if it's the correct palette (looks cool though). It's clearly just the first image but slightly changed about to have a different barrel and no fore sight on it. Also the palette I got this from puts a load of like red colours around it, so I don't know if it is correct at all. Basically mysterious. 

I'm still trying to find the briefcase graphic(s), and also noticed there is a nice new explosives image, which I think I have found in the sheets, but can't find the palette for yet.

**Comment by Moburma** [comment]
Finally ripped the briefcases. Not that exciting as they are quite close to the PC ones, but for completionists:

![](https://i.ibb.co/9mZMFZLs/silverbriefcase.png)
![](https://i.ibb.co/RkG7rNCS/greenbriefcase.png)

I also successfully inserted many into the PC game. Here are the Cerberus IFFs:
![](https://i.ibb.co/HDKH2vWt/Desktop-Screenshot-2025-06-29-23-14-22-55.png)

I will try and make a complete set of files for these if anyone wants them. You have to be careful as exceeding the original graphic's width causes glitches (needs the .ani files tweaked I guess). 
The new tree looks weird, as it's taller than the original so it's not anchored against the floor, it extends below it, which is evident on rotation. That said the Cerberus IFF alone is enough for me, it's better than the terrible original graphic.

**Comment by Moburma** [comment]
Got the bombs as well:
![](https://i.ibb.co/TD7bRFnZ/bomb1-f.png)
![](https://i.ibb.co/99XDmf0t/bomb2-f2.png)

They actually work in a different way in the PS1 version. The ion mine and explosives share the two graphics, but the ion mine quickly flashes between the two giving it a kind of flickering blue look, whereas the explosives shows mainly the orange variant, but flashes briefly to the blue one as it counts down, the pace of flashing increasing as it reaches zero. These graphics don't work well in the PC version as they are slightly wider causing it to glitch, but also there simply isn't the palette depth to accurately render them in darkness, etc.

**Comment by mefistotelis** [comment]
One thing from these which sounds like a real improvement would be for High Explosive to use the flashing with increasing frequency. The explosive does lack a visual queue on what is going on.

Though I think a 4-frame animation would look a lot better than blinking between just two images.

Not only it would inform how much time is left, but also allow to differentiate armed and secured explosives.
(though it's still strange to have red light on secured ones)

**Reply by Moburma** [reply]
Yes, it's one of those obviously good ideas that I couldn't remember if the PC one did or not when I noticed it, and assumed it did. Sitting down and checking and of course it actually doesn't. They genuinely did good work on the PS1 version and it has some nice improvements like that. There are also red coloured light effects on the bombs and the Cerberus IFF to draw the eye visually to them as well.
